packages/button/src/docs/Component.stories.mdx

```diff
@@ -17,6 +17,7 @@ import variantStyles from '!!raw-loader!../styles/variant.module.css';
   argTypes={{
     children: {
       control: 'text',
+      type: { name: 'string', required: false },
```

Comment:
why are we setting type of children as string here? isn't that always ReactNode in our components? 

`children?: React.ReactNode;
`
https://github.com/hey-car/hey-ui/blob/08798f872d1bc2a4f906ff721706568430414cc7/packages/icons/src/SvgIcon.types.tsx#L7